src/plugins/gs-flatpak.c

```diff
@@ -1617,3 +1617,14 @@ gs_flatpak_new (GsPlugin *plugin, GsFlatpakScope scope)
 	self->plugin = g_object_ref (plugin);
 	return GS_FLATPAK (self);
 }
+
+const char *
+gs_flatpak_get_prefix (GsFlatpak *self)
+{
+	if (self->scope == GS_FLATPAK_SCOPE_USER)
+		return GS_FLATPAK_USER_PREFIX;
+	else if (self->scope == GS_FLATPAK_SCOPE_SYSTEM)
+		return GS_FLATPAK_SYSTEM_PREFIX;
+
+	return "";
```

Comment:
If there are no other scopes, should use a `g_assert_if_reached()` or similar perhaps?